samples/demo/app.bicep

```diff
@@ -3,12 +3,14 @@ import radius as radius
 param application string
 param environment string
 
+param image string = 'ghcr.io/radius-project/samples/demo:latest'
```

Comment:
I had to add these image parameters to the actual sample files. Is there a better way?